docs/components/modeler/bpmn/escalation-events/escalation-events.md

```diff
@@ -0,0 +1,73 @@
+---
+id: escalation-events
+title: "Escalation events"
+description: "Escalation events are used to escalate part of process execution to a higher flow scope."
+---
+
+Escalation events are events which reference a named escalation. They are used to communicate to a higher flow scope.
+Unlike an error, an escalation event is non-critical and execution continues at the location of throwing.
+
+![The process reached an escalation event. The escalation gets caught in a higher flow scope. As the escalation throw event is non-critical, the outgoing sequence flow of this event is taken.](assets/escalation-events.png)
+
+The example above shows the execution of an escalation event. The process reaches the `Throw` event.
+This will throw an escalation to a higher flow scope. The escalation is caught by the `Catch` event. As escalation events
+are non-critical, the outgoing sequence flows of `Throw` and `Catch` are both taken.
+
+## Defining an escalation
+
+In BPMN, an `escalation event` references an `escalation`. Escalations can be referenced by one or more escalation events.
+
+An escalation must define an `escalationCode`. The value of this `escalationCode` is used to determine which catch event
+can catch the thrown escalation.
+
+For throwing escalation events it is possible to define the `escalationCode` as an `expression`. When the event is reached the expression gets evaluated.
+An escalation with the result of this expression is thrown. If no expression is used the statically defined `escalationCode` is used.
+
+For catching escalation events it is not possible to use an `expression`. The `escalationCode` must always be a static value.
+Alternatively, the `escalationCode` can be left empty. A catch event with an empty `escalationCode` will catch **all** thrown escalations.
+
+## Throwing the escalation
+
+An escalation can be thrown by an escalation end event, or by an intermediate escalation throw event. Escalation events
+are non-critical. This means that if the throwing event has any outgoing sequence flows, they will be taken.
+
+## Catching the escalation
+
+An escalation can be caught using a boundary event, or using an event subprocess. It is caught by one catch event at most.
+This will be the catch event in the nearest flow scope.
```

Comment:
:thinking: Should we clarify that it's the nearest **parent** flow scope?

```suggestion
This will be the catch event in the nearest parent flow scope.
```

Reply:
I guess we should. I figured it was obvious, but for users reading this it might not be.